nautobot/dcim/migrations/0063_device_access_point_group.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.15 on 2024-10-05 21:20
```

Comment:
We'll want to re-generate migrations before merging into next so that we introduce as few migrations as possible.